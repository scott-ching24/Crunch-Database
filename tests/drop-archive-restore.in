set a 1 2
set b 2 3
snapshot
set c 3 4
set d 5 6
snapshot
archive 1 "backup.txt"
drop 1
list snapshots
purge a
purge b
restore "backup.txt"
list entries
list snapshots
bye
