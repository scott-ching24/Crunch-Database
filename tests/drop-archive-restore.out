> ok

> ok

> saved as snapshot 1

> ok

> ok

> saved as snapshot 2

> ok

> ok

> 2

> ok

> ok

> ok

> b [2 3]
a [1 2]

> no snapshots

> bye
